examples/create_upload_tutorial.py

```diff
@@ -7,39 +7,70 @@
 import numpy as np
 import openml
 import sklearn.datasets
+from scipy.sparse import coo_matrix
+from openml.datasets.functions import create_dataset
 
 ############################################################################
-# For this example we will upload to the test server to not  pollute the live server with countless copies of the same dataset.
+# For this tutorial we will upload to the test server to not  pollute the live
+# server with countless copies of the same dataset.
 openml.config.server = 'https://test.openml.org/api/v1/xml'
 
 ############################################################################
-# Prepare the data
-# ^^^^^^^^^^^^^^^^
+# The dataset that you upload to OpenML can be:
+#
+# * A numpy array.
+# * A list of lists.
+# * A sparse matrix
+
+############################################################################
+# Dataset is a numpy array
+# ========================
+#
+# Prepare dataset
+# ^^^^^^^^^^^^^^^
 # Load an example dataset from scikit-learn which we will upload to OpenML.org via the API.
+
 breast_cancer = sklearn.datasets.load_breast_cancer()
 name = 'BreastCancer(scikit-learn)'
-X = breast_cancer.data
+x = breast_cancer.data
 y = breast_cancer.target
+target_names = breast_cancer.target_names
+y = np.array([target_names[i] for i in y])
```

Comment:
I would force the array to be `object` dtype since this is containing strings.

Reply:
just noticed, I would agree with this, however instead we are going to go for another dataset.